src/main/java/com/box/sdk/BoxDeserializationException.java

```diff
@@ -0,0 +1,50 @@
+package com.box.sdk;
+
+/**
+ * A class representing exceptions caused from deserializing errors.
+ */
+public class BoxDeserializationException extends RuntimeException {
+    private String fieldName;
+    private String fieldValue;
+    private String errorMessage;
+
+    /**
+     * Initializes the BoxDeserializationException class.
+     *
+     * @param member the key of the json member the deserialization occurred on.
+     * @param value the value of the json member the deserialization occurred on.
+     * @param className the name of the class the deserialization occurred on.
+     * @param e the throwable cause for the exception.
+     */
+    public BoxDeserializationException(String member, String value, String className, Exception e) {
+        super(e);
```

Comment:
There's an `Exception` [constructor overload][ex-con] that takes both the message and the cause; it's probably better to use that than the `errorMessage` property, since that's not a standard exception field

[ex-con]: https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html#Exception(java.lang.String,%20java.lang.Throwable)